code/modules/antagonists/gang/gang.dm

```diff
@@ -88,61 +90,56 @@
+	to_chat(owner.current, "<B><font size=6 color=red>You are still a team-oriented antagonist! Do what is best for your gang and it's objective.</font></B>")
+	var/assigned_objective = my_gang.current_theme.gang_objectives[src.type]
+	if(!assigned_objective)
+		assigned_objective = "ERROR, FILE A REPORT WITH THIS INFO: Gang Name: [gang_name], Theme Name: [my_gang.current_theme.name]"
+	to_chat(owner.current, "<B><font size=4 color=red>Family's Objective:</B> [assigned_objective]</font>")
 
 /datum/antagonist/gang/apply_innate_effects(mob/living/mob_override)
 	..()
 	package_spawner.Grant(owner.current)
 	package_spawner.my_gang_datum = src
 	var/mob/living/M = mob_override || owner.current
 	add_antag_hud(antag_hud_type, antag_hud_name, M)
-	if(M.hud_used)
-		var/datum/hud/H = M.hud_used
-		var/atom/movable/screen/wanted/giving_wanted_lvl = new /atom/movable/screen/wanted()
-		H.wanted_lvl = giving_wanted_lvl
-		giving_wanted_lvl.hud = H
-		H.infodisplay += giving_wanted_lvl
-		H.mymob.client.screen += giving_wanted_lvl
 
 /datum/antagonist/gang/remove_innate_effects(mob/living/mob_override)
 	package_spawner.Remove(owner.current)
 	var/mob/living/M = mob_override || owner.current
 	remove_antag_hud(antag_hud_type, M)
-	if(M.hud_used)
-		var/datum/hud/H = M.hud_used
-		H.infodisplay -= H.wanted_lvl
-		QDEL_NULL(H.wanted_lvl)
 	..()
 
-/// Passes points gained to the family team.
-/datum/antagonist/gang/proc/add_gang_points(points_to_add)
-	if(my_gang)
-		my_gang.adjust_points(points_to_add)
-
-/// Determines if a gang has completed their special objective.
-/datum/antagonist/gang/proc/check_gang_objective()
-	return TRUE
-
 /// Gives a gangster their equipment in their backpack and / or pockets.
 /datum/antagonist/gang/proc/equip_gangster_in_inventory()
 	if(istype(owner.current, /mob/living/carbon/human))
-		for(var/C in my_gang.free_clothes)
-			var/obj/O = new C(owner.current)
-			var/list/slots = list (
-				"backpack" = ITEM_SLOT_BACKPACK,
-				"left pocket" = ITEM_SLOT_LPOCKET,
-				"right pocket" = ITEM_SLOT_RPOCKET
-			)
+		var/list/slots = list (
+			"backpack" = ITEM_SLOT_BACKPACK,
+			"left pocket" = ITEM_SLOT_LPOCKET,
+			"right pocket" = ITEM_SLOT_RPOCKET,
+			"hands" = ITEM_SLOT_HANDS
+		)
+		for(var/clothing in my_gang.free_clothes)
+			var/obj/O = new clothing(owner.current)
 			var/mob/living/carbon/human/H = owner.current
 			var/equipped = H.equip_in_one_of_slots(O, slots)
 			if(!equipped)
-				to_chat(owner.current, "Unfortunately, you could not bring your [O] to this shift. You will need to find one.")
-				qdel(O)
-
+				to_chat(owner.current, "Your [O] has been placed at your feet.")
+				O.forceMove(get_turf(H))
+		for(var/bonus_item in my_gang.current_theme.bonus_items)
+			var/obj/O = new bonus_item(owner.current)
+			var/mob/living/carbon/human/H = owner.current
+			var/equipped = H.equip_in_one_of_slots(O, slots)
+			if(!equipped)
+				to_chat(owner.current, "Your [O] has been placed at your feet.")
```

Comment:
You need to use `O.name`, otherwise it'll show as "Your the gang package has been placed"

Reply:
Also wrap them all in `span_notice`.